proto/cmp/services/activity/v1alpha/activity_types.proto

```diff
@@ -0,0 +1,235 @@
+
+    // Zones
+    repeated ActivityZoneInfo zones = 4;
+
+    // Descriptions with different languages
+    repeated cmp.types.v1alpha.LocalizedDescriptionSet descriptions = 5;
+
+    // Coordinates
+    cmp.types.v1alpha.Coordinate position =6;
+
+    // Activity Features
+    repeated ActivityFeature features = 7;
+
+    // Tags
+    repeated ActivityTag tags = 8;
+
+    // Languages
+    repeated cmp.types.v1alpha.Language languages = 9;
+
+    // Contact Info; Address, Email, Phone, Urls
+    ActivityContactInfo contact_info = 10;
+
+    // Images
+    repeated cmp.types.v1alpha.Image images = 11;
+
+    // Videos
+    repeated cmp.types.v1alpha.Video videos = 12;
+
+}
+
+message ActivityProductInfoDetails {
+  // Context for Inventory system concepts that need to be included in an info response,
+  // like an OwnerCode, PTC_OfferParameters, Tax codes, Disclosure RefID, etc. or a
+  // serialized combination of these codes.
+  string context = 1;
+
+  // Errors encountered in the generation of the response. Standardized error list
+  // to be created.
+  repeated string errors = 2;
+
+  // Warnings encountered in the generation of the response. This section might include
+  // information about fields not processed due to incompatibilities with the
+  // inventory system
+  repeated string warnings = 3;
+
+  // Supplier code from the Inventory System for this activity response.
+  string supplier_unit_code = 4;
+
+  // An identifier for external sessions, aiding in tracking and continuity across
+  // sessions.
+  string external_session_id = 5;
+
+  // Activity Category Code
+  string category_code = 6;
+
+  // Category name which describes the category_code
+  string category_name = 7;
+
+  // Type Code
+  string type_code = 8;
+
+  // Type name which describes type_code
+  string type_name = 9;
+
+  // Supplier specific code, matching the supplier code in ProductList and ProductInfo
+  // messages
+  cmp.types.v1alpha.SupplierProductCode supplier_code = 10;
+
+  // Supplier Name
+  string supplier_code_name = 11;
+
+  // product_code
+  cmp.types.v1alpha.ProductCode product_code = 12;
+
+  // Duration range (min, max) of the activity in minutes
+  cmp.types.v1alpha.DurationRange duration_range = 13;
+
+  // Time given for confirmation before the offer expires
+  cmp.types.v1alpha.Duration max_confirmation_duration = 14; 
+
+  // Allow Free Sale
+  bool allow_free_sale = 15;
+
+  // The ability to confirm instantly
+  bool instant_confirmation = 16;
+
+  // Can be delivered instantly
+  bool instant_delivery = 17;
+
+  // Check availability
+  bool availability_required = 18;
+  
+  // Availability Type
+  string availability_type = 19;
+
+  // Delivery Formats; such as QR Code, NFT, Ticket, ... etc
+  repeated string delivery_formats = 20;
+
+  // Delivery Methods; such as Email, SMS, Post, ... etc
+  repeated string delivery_methods = 21;
+
+  // Redemption method
+  string redemption_method = 22;
+
+}
+
+
+message ActivityUnitInfo {
```

Comment:
```suggestion
message ActivityUnit {
```
Same here and other types below.

Just `ActivityUnit`? Wdyt?

Reply:
OK